docs/adr/ADR-025-release-management-and-versioning-policy.md

```
@@ -4,28 +4,39 @@
 accepted
 
+Updated 2026-07-04: EventCore now has nine published crates at 1.x, the
+release workflow runs in GitHub Actions, and full lockstep versioning is the
+implemented policy.
+
 ## Context
 
-EventCore is a Rust workspace with six published crates distributed via crates.io:
+EventCore is a Rust workspace with nine published crates distributed via crates.io:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)